gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=0.10.8-SNAPSHOT
-ballerinaLangVersion=2201.4.0-20221212-011800-9715f15d
+ballerinaLangVersion=2201.3.1
```

Comment:
Shall we stick to update 4 snapshot versions, since we'll be publishing the new changes along with U4 release?

Reply:
yeah sure